### PR TITLE
[1LP][RFR]map_tags navigation and crud fixes

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Fixed map tags navigation, update and delete methods needed for new tests(https://github.com/ManageIQ/integration_tests/pull/5036)

### CODE REVIEW

File: cfme/configure/configuration/region_settings.py [hunk 1]
```diff
@@ -355,7 +355,7 @@ def create(self, cancel=False):
             flash_message = 'Add of new Container Label Tag Mapping was cancelled by the user'
         else:
             view.add_button.click()
-            flash_message = 'Container Label Tag Mapping "{}" was saved'.format(self.label)
+            flash_message = 'Container Label Tag Mapping "{}" was added'.format(self.label)
```
Comment (mkoura) [inline cfme/configure/configuration/region_settings.py]: Is this the same for 5.8+ and 5.7?

File: cfme/configure/configuration/region_settings.py [hunk 2]
```diff
@@ -368,27 +368,32 @@ def update(self, updates, cancel=False):
...
+            )
         else:
             view.save_button.click()
             flash_message = 'Container Label Tag Mapping "{}" was saved'.format(self.label)
 
         view = self.create_view(MapTagsAllView, override=updates)
         view.flash.assert_success_message(flash_message)
 
-    def delete(self, cancel=True):
+    def delete(self, cancel=False):
```
Comment (mkoura) [inline cfme/configure/configuration/region_settings.py]: Will this change of default value affect existing tests?

File: cfme/configure/configuration/region_settings.py (hunk 1 above)
Comment (rlbabyuk) [inline cfme/configure/configuration/region_settings.py]: yes the same for both versions, my typo correction

File: cfme/configure/configuration/region_settings.py (hunk 2 above)
Comment (rlbabyuk) [inline cfme/configure/configuration/region_settings.py]: no, it won't affect, there are no existing tests for map tags now. But they are just in progress in https://github.com/ManageIQ/integration_tests/pull/5036 PR

**Pull request merged**